src/test/java/org/sonarsource/sonarlint/ls/mediumtests/JavaMediumTests.java

```diff
@@ -232,9 +232,9 @@ void analyzeSimpleJavaFilePassVmClasspath() throws Exception {
     assertThat(client.logs)
       .extracting(withoutTimestamp())
       .containsSubsequence(
-        "[Debug] [sonarlint : sonarlint-analysis-engine] Property 'sonar.java.jdkHome' set with: " + currentJdkHome,
-        "[Debug] [sonarlint : sonarlint-analysis-engine] Property 'sonar.java.jdkHome' resolved with:" + System.lineSeparator() + "[" + jrtFsJarPath + "]",
-        "[Debug] [sonarlint : sonarlint-analysis-engine] Property 'sonar.java.libraries' resolved with:" + System.lineSeparator() + "[" + jrtFsJarPath + "]");
+        "[Debug] [org.sonar.java.classpath.AbstractClasspath : sonarlint-analysis-engine] Property 'sonar.java.jdkHome' set with: " + currentJdkHome,
```

Comment:
Is there still value in logging the logger name? Maybe we could only keep the thread name. But it can be for the future.